routes/api/v1/sessions.js

```diff
@@ -0,0 +1,40 @@
+var express = require("express");
+var router = express.Router();
+var User = require('../../../models').User;
+
+const bcrypt = require('bcrypt');
+
+var checkPassword = function(password, hash){
```

Comment:
Looks like `checkPassword()` isn't being utilized. In line 16 within `verifyLogin()` you've got very similar functionality, so you could continue just using that, or use `checkPassword()` there.

Reply:
You're absolutely right. I got stuck in promise hell, so moved that back into verifyLogin after having previously moved it out, then forgot to get rid of the now-unused function.